src/js/zql/toZql.ts

```diff
@@ -2,7 +2,28 @@ import isString from "lodash/isString"
 import {zed} from "zealot"
 import {isStringy} from "zealot/zed"
 
+const needsQuotes = (fieldName: string) => !/^[a-zA-Z_$][\w]*$/.test(fieldName)
+
+// TODO: Mason - make field type a 'zed.Field' once it has path getter
+export const toFieldPath = (field: any) => {
```

Comment:
Once the `zed.Field` has been modified, I'm planning on changing the test I included to test `toZql` instead and no longer export this function. I just needed a fast way to prove this works.